shared/cfg/guest-os/Linux/Fedora/25.x86_64.cfg

```diff
@@ -0,0 +1,27 @@
+- 25.x86_64:
+    image_name = images/f25-64
+    vm_arch_name = x86_64
+    os_variant = fedora25
+    # Installation works fine with mem=1024 on methods such as cdrom
+    # but fails ("No space left on device") with methods such as url.
+    # anyway, let's use a safe default
+    mem = 2048
```

Comment:
We could, but there's prior art on setting those at the guest level.  I'd believe (but have not checked) that guest-os/*cfg are included before the test configuration files.